notebooks/depth_filters.ipynb

```diff
@@ -576,7 +576,7 @@
    "metadata": {},
    "source": [
     "\n",
-    "> **Want to learn more?** Visit [realsense.intel.com](http://realsense.intel.com) and [github.com/IntelRealSense](http://www.github.com/IntelRealSense/librealsense)"
+    "> **Want to learn more?** Visit [realsense.intel.com](https://realsense.intel.com) and [github.com/IntelRealSense](https://www.github.com/IntelRealSense/librealsense)"
```

Comment:
http~~s~~://www.github.com/  should probably not be affected
Also verify https://realsense.intel.com - this was not in the scope